monitor/internal/pod/controller.go

```diff
@@ -101,8 +103,9 @@ var _ reconcile.Reconciler = &ReconcilePod{}
 // them for real deletion in the Kubernetes API. Once an object is gone, we will
 // send down destroy events to trireme.
 type DeleteEvent struct {
-	NativeID string
-	Key      client.ObjectKey
+	NativeID  string
```

Comment:
Can we also rename here NativeID to PodUID? And Maybe Key to NamespacedName?

Again, just for clarity in the future as we've been going forth and back on the NativeIDs